man/systemctl.xml

```diff
@@ -1245,6 +1245,28 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
         </varlistentry>
 
         <varlistentry>
+          <term><command>revert <replaceable>NAME</replaceable>...</command></term>
```

Comment:
I'm not crazy about the name. I think for any git user "revert" is forever tied to reverting a single commit, and in case of unit files, when I hear "revert", I think about a undoing the last systemctl operation. I think "reset" would be a better verb.